.editorconfig

```diff
@@ -0,0 +1,39 @@
+# http://editorconfig.org
```

Comment:
👍 

Would it be worth popping an additional bit in our 'setting up a repo' guidance doc about this file? I guess this would be used for all new repos as standard now?

Reply:
Yep, makes sense, updated 👍

Reply:
Sorry, I meant in the "eQ: Setting up new GitHub repo" doc. Although it's also good that it's in the readme for this repo too 👍

Reply:
Yep, will add that in 👍

Reply:
Added in here: https://confluence.ons.gov.uk/display/SDC/eQ%3A+Setting+up+new+GitHub+repo